src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/McpServer/McpServer-CSharp/Tools/RandomNumberTools.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel;
+using ModelContextProtocol.Server;
+
+/// <summary>
+/// Sample MCP tools for demonstration purposes.
+/// These tools can be invoked by MCP clients to perform various operations.
+/// </summary>
+internal class RandomNumberTools
+{
+    /// <summary>
+    /// Returns a random number between 1 and the maximum number allowed by the tool (inclusive).
```

Comment:
This comment is no longer accurate. I'm also not sure an XML comment on this is helpful.